memory/src/main/java/org/opensearch/ml/memory/index/InteractionsIndex.java

```diff
@@ -421,17 +508,21 @@ public void getInteraction(String conversationId, String interactionId, ActionLi
                     client
                         .admin()
                         .indices()
-                        .refresh(Requests.refreshRequest(INTERACTIONS_INDEX_NAME), ActionListener.wrap(refreshResponse -> {
-                            client.get(request, al);
-                        }, e -> {
-                            log.error("Failed to refresh interactions index during get interaction ", e);
-                            internalListener.onFailure(e);
-                        }));
+                        .refresh(
```

Comment:
Same here, why refresh before getting ? How about change to refresh for write request (create/update)?